src/config.ml

```diff
@@ -1642,6 +1658,23 @@ let a_remote :
 
 let a_remote_entry = a_remote
 
+let a_push =
+  string "push" *> a_whitespace *> commit *> a_single_param
+  >>= fun push_option ->
+  (* the OpenVPN man page says only a subset of options can be pushed.
+     This is however misleading as the server can push what it pleases.
+     Only, the client will filter out unaccepted options.
+
+     The only constraint seems to be the option can't contain commas as
+     comma will be used as a separator when pushing options. *)
+  if String.contains push_option ',' then
+    let msg =
+      Fmt.str "Push option contains illegal comma (',') in string: %S"
+        push_option
+    in
+    return (`Ignored msg)
```

Comment:
ignored or error? I'm a fan of fail early and fail hard.

Reply:
As I recall I found that this is the behavior of OpenVPN. We may reconsider this...

Reply:
addressed in https://github.com/robur-coop/miragevpn/commit/cd7d999321e13993862af649977689aa96a7e114